app/models/data_request_area.rb

```diff
@@ -0,0 +1,68 @@
+# == Schema Information
+#
+# Table name: data_request_areas
+#
+#  id                     :bigint           not null, primary key
+#  case_id                :bigint           not null
+#  user_id                :bigint           not null
+#  contact_id             :bigint
+#  data_request_area_type :enum             not null
+#  location               :string
+#  created_at             :datetime         not null
+#  updated_at             :datetime         not null
+#
+class DataRequestArea < ApplicationRecord
+  belongs_to :offender_sar_case, class_name: "Case::Base", foreign_key: "case_id"
+  belongs_to :user
+  belongs_to :contact
+  has_many :data_requests
+  has_one :commissioning_document
+  has_many :data_request_emails
+
+  validates :data_request_area_type, presence: true
+  validates :offender_sar_case, presence: true
+  validates :user, presence: true
+
+  before_validation :clean_attributes
+
+  #TODO this should call when attempting to send commissioning doc
+  # validate  :validate_location
+
+  attribute :data_request_default_area, default: ""
+  attribute :data_request_area_type
+  attribute :location
```

Comment:
What's the purpose of these?

Reply:
I removed these but then remembered that without `attribute :data_request_default_area, default: ""` and a hidden field within app/views/cases/data_request_areas/new.html.slim causes a parameter missing error if no radio button is selected

Instead of this default value I could change the create_params like this:

 def create_params
      params.fetch(:data_request_area, {}).permit(:data_request_area_type)
    end